README.rst

```diff
@@ -6,12 +6,12 @@ Photutils
 
 |CI Status| |Codecov Status| |Latest RTD Status|
 
-Photutils is an `Astropy`_ package for detection and photometry of
+Photutils is an `Astropy`_ package for detecting and performing photometry of
```

Comment:
I've removed this because it is unrelated to fixing the outdated installation link.  I'm going to update and expand this description in a separate PR.